src/System.Net.Http/src/Resources/Strings.resx

```diff
@@ -354,4 +354,7 @@
   <data name="net_http_unix_invalid_response" xml:space="preserve">
     <value>The server returned an invalid or unrecognized response.</value>
   </data>
-</root>
+  <data name="Argument_BufferNotLongEnough" xml:space="preserve">
+    <value>The buffer was not long enough.</value>
+  </data>
+</root>
```

Comment:
Not sure I understand how VS messes this up.  But you should fix any files you touched.